packages/components/src/range-control/index.tsx

```diff
@@ -212,6 +213,7 @@ function UnforwardedRangeControl< IconProps = unknown >(
 
 	return (
 		<BaseControl
```

Comment:
@ciampo There is an ever so slight space here that I can only resolve by setting a `line-height: 0` on `.components-base-control__field`, and I don't know what is happening 😂 Let me know if you have any ideas.

![Small space after the input field](https://user-images.githubusercontent.com/555336/183429306-41b5434d-2a6c-498b-b4b0-20a1591f83ed.png)

Reply:
That's the typical "white space" issue that happens around inline elements — there are different ways to fix, including setting `font-size: 0` or `line-height: 0` on `.components-base-control__field`.

But in my opinion those are just "patchy" fixes — a better way to handle it is to avoid having inline elements, which cause the white space in the first place.

The easiest fix would be to change `RangeControl`'s [`Root` element](https://github.com/WordPress/gutenberg/blob/a862658f8a8b94e1be4e2c53ebc4c12fdf02d575/packages/components/src/range-control/styles/range-control-styles.ts#L32) from `display: inline-flex` to `display: flex`.

In case there was a reason for that `Root` element to be `inline-flex` and we don't want to touch it, another alternative is to set `display: flex; flex-direction: column` on `.components-base-control__field` (also in this scenario, we'd need to check that this change doesn't introduce any regression).

Bonus round: for both proposed solutions, we could use `HStack` or `VStack` instead!

Reply:
I had so much trouble finding the culprit, thank you 😂 

> The easiest fix would be to change `RangeControl`'s [`Root` element](https://github.com/WordPress/gutenberg/blob/a862658f8a8b94e1be4e2c53ebc4c12fdf02d575/packages/components/src/range-control/styles/range-control-styles.ts#L32) from `display: inline-flex` to `display: flex`.

I went with this fix — it looks safe from what I could gather.